### PR TITLE
Rollback xcodeproj conf #no-public-changes

### PR DESCRIPTION
**Pull request by tbaranes**
- Two spaces indentation by default
- SwiftLint scripts: I duplicated the .swiftlint.yml to make sure it takes effects for the frameworks and the demo app, a better solution would be to add only one in the root directory, but I don't know how makes this working right now

### CODE REVIEW

**Comment by tbaranes** [comment]
Seems like Danger is broken as well 🤔

**Comment by tbaranes** [comment]
Update: Add `IBAnimatable.framework` as embedded binaries in the example app xcodeproj to ensure it will be compile each time (if necessary)

File: IBAnimatable/IBAnimatable.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -418,7 +418,10 @@
 				E27FD0041ECD79DD00F74840 /* IBAnimatableTests */,
 				E27FCFF81ECD79DD00F74840 /* Products */,
 			);
+			indentWidth = 2;
```
Comment (JakeLin) [inline IBAnimatable/IBAnimatable.xcodeproj/project.pbxproj:421]: Good.

**Review by JakeLin — APPROVED** [review]
👍

**Pull request merged**